circle.yml

```diff
@@ -0,0 +1,10 @@
+## Customize dependencies
+dependencies:
+  override:
+    - npm prune
```

Comment:
@vdh Yeah, I mean both `prune` and `update`
Did you know you could clear the cache: https://circleci.com/docs/how-cache-works/#clearing-the-cache